docs/rules/no-window-top.md

```diff
@@ -0,0 +1,28 @@
+# Disallow window.top use (no-window-top)
+
+Lighting Locker vNext prohibits access to `window.top` to prevent sandboxed code
+from accessing the non-proxied Blue `window`. When Locker vNext is released
+`window.top` will be `null` or transpiled to the Red proxied Blue `window`.
+
+The following `window.top` references are prohibited or fixable to `window`:
```

Comment:
I am don't know enough about Locker VNext but, I think the underlying assumption is that in Locker the `window` object in the current evaluation context is always the topmost `window` in the hierarchy due to sandboxing. I think this should be explicitly stated here to explain how this rule works.